v1/topdown/http.go

```diff
@@ -1006,8 +1006,10 @@ func insertIntoHTTPSendInterQueryCache(bctx BuiltinContext, key ast.Value, resp
 }
 
 func createKeys() {
+	ast.InternedStringTerm.Store(allowedKeyNames[:]...)
+
 	for _, element := range allowedKeyNames {
-		term := ast.StringTerm(element)
+		term := ast.InternedStringTerm.Get(element)
```

Comment:
If we were to add an upper cap to the interning cache, this would need to use `GetOrCreate()` as we'd not be guaranteed a non-nil value.